dmapiclient/search.py

```diff
@@ -112,11 +112,11 @@ def index(
         url = '/{}/{}/{}'.format(index_name, doc_type, object_id)
         return self._put(url, data={'document': serialized_object}, client_wait_for_response=client_wait_for_response)
 
-    def delete(self, index, service_id):
+    def delete(self, index, service_id, *, client_wait_for_response: bool = True):
```

Comment:
Is there any reason to have this called `client_wait_for_response` rather than `wait_for_index` like `update_service_status`?

Reply:
Yes, the distinction is that the argument _here_, like the one above it on `index()` is an instruction to the apiclient itself. `wait_for_index` is an instruction to the `api` to tell _it_ to use a `client_wait_for_response` argument on _its_ apiclient when _it_ is doing the daisychained call to the search api.